src/Core/src/Platform/Android/MauiTextView.cs

```diff
@@ -20,21 +21,16 @@ internal override void OnLayoutFormatted(bool changed, int l, int t, int r, int
 		}
 		protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 		{
-			if (MeasureSpec.GetMode(widthMeasureSpec) == MeasureSpecMode.AtMost && Layout is not null)
+			if (MeasureSpec.GetMode(widthMeasureSpec) == MeasureSpecMode.AtMost && !string.IsNullOrEmpty(Text))
```

Comment:
You check` !string.IsNullOrEmpty(Text)`, but if Text is empty later, will the component resize correctly?

Reply:
I have verified this by dynamically setting the text to an empty string (string.Empty); it resizes correctly, just as it did before the fix.